includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -20,13 +20,16 @@ public function init() {
 		add_action( 'edit_post', array( $this, 'track_product_updated' ), 10, 2 );
 		add_action( 'transition_post_status', array( $this, 'track_product_published' ), 10, 3 );
 		add_action( 'created_product_cat', array( $this, 'track_product_category_created' ) );
+		add_action( 'woocommerce_variation_set_stock', array( $this, 'track_product_stock_level_set' ), 10, 1 );
+		add_action( 'woocommerce_product_set_stock', array( $this, 'track_product_stock_level_set' ), 10, 1 );
+		add_action( 'add_meta_boxes_product', array( $this, 'track_product_updated_client_side' ), 10, 1 );
 	}
 
 	/**
-	 * Send a Tracks event when a product is updated.
+	 * Send some Tracks events when a product is updated.
```

Comment:
This can go back to its original description.

Reply:
Done